fail2ban/tests/utils.py

```diff
@@ -50,6 +54,10 @@
 	else:
 		CONFIG_DIR = '/etc/fail2ban'
 
+# In not installed env (setup, test-cases) use fail2ban modules from main directory:
+if 1 or os.environ.get('PYTHONPATH', None) is None:
```

Comment:
No, in the test case we should always add the current root path as PYTHONPATH variable, because the f2b-modules are not installed and in forked processes it is wrong otherwise. I've just forgot to remove `if 1 or ...` line.

About others of your comments: thanks for review, I'll take a look tomorrow...